src/v3/src/transformer/transaction/transform.ts

```diff
@@ -50,6 +61,54 @@ const transformRemediationNameForIdp: TransformStepFnWithOptions = (options) =>
   return formbag;
 };
 
+/**
+ * To support `idps` configuration in OIE.  Adds IDP buttons from widget config.
+ * https://github.com/okta/okta-signin-widget#openid-connect
+ */
+const injectIdPConfigButtonToRemediation: TransformStepFnWithOptions = (options) => (formbag) => {
```

Comment:
This is for the openid-connect feature where idp buttons can be added in the Widget Config file.  More info on this can be found here: 
https://github.com/okta/okta-signin-widget/blob/b911ee7b11ecd5938726e651c1ad937a597bbdb8/docs/classic.md?plain=1#LL1295C7-L1295C7
https://github.com/okta/okta-signin-widget/pull/1237